pkg/services/object/get/assemble.go

```diff
@@ -168,7 +168,7 @@ func (exec *execCtx) overtakePayloadDirectly(children []oid.ID, rngs []objectSDK
 			return
 		}
 
-		if ok := exec.writeObjectPayload(child); !ok {
+		if ok := exec.writeObjectPayload(child, nil); !ok {
```

Comment:
Why not streaming here? `getChild()` should be adapted for this as well. This optimization is very much relevant for big objects that stream GBs of data.